src/main/java/org/broadinstitute/consent/http/resources/DacResource.java

```diff
@@ -40,8 +42,15 @@ public DacResource(DacService dacService) {
     @GET
     @Produces("application/json")
     @RolesAllowed({ADMIN, MEMBER, CHAIRPERSON})
-    public Response findAll() {
-        List<Dac> dacs = dacService.findAllDacsWithMembers();
+    public Response findAll(@Auth AuthUser authUser, @QueryParam("withUsers") Optional<Boolean> withUsers) {
+        List<Dac> dacs = new ArrayList<>();
+        final Boolean includeUsers = withUsers.isPresent() ? withUsers.get() : true;
+        if (includeUsers) {
+            dacs.addAll(dacService.findAllDacsWithMembers());
+        }
+        else {
+            dacs.addAll(dacService.findDacsByUser(authUser));
+        }
```

Comment:
Thank you for this change, I think it simplifies the overall work this class does.